m4opt/models/bandpass/bandpass.py

```
@@ -0,0 +1,115 @@
+    ---> gauss1d.result.input_units_equivalencies = {'x': u.spectral()}
+
+    will work with other `m4opt` models, but
+
+    ---> gauss1d = Gaussian1D(amplitude=0.5, mean=6000 * u.erg, stddev=500 * u.erg)
+
+    will not.
+    """
+
+    @classmethod
+    def from_table(cls, points, lookup_table):
+        """
+        Initializes bandpass from user-defined table.
+
+        points : 1D array of `astropy.Quantity` (wavelength or frequency)
+        lookup_table : 1D array of bandpass response
+        """
+
+        unit_type = get_physical_type(points)
+        if unit_type != "length" and unit_type != "frequency":
+            raise AttributeError("Input array points does not have " +
+                                 "appropriate wavelength or frequency " +
+                                 "units.")
+
+        result = Tabular1D(points, lookup_table*u.dimensionless_unscaled)
+        result.input_units_equivalencies = (BaseBandpass.
+                                            input_units_equivalencies)
+        return result
+
+    @classmethod
+    def from_name(cls, filter_name):
+        """
+        Initializes bandpass from tynt package.
+        """
+        if tynt_filters is not None:
+            if filter_name not in tynt_filters:
+                raise ValueError("Invalid filter {0}. ".format(filter_name) +
+                                 "See 'm4opt.bandpass.tynt_filters' for " +
+                                 "available options.")
+
+            filter = FilterGenerator().reconstruct(filter_name)
+            result = Tabular1D(filter.wavelength,
+                               filter.transmittance*u.dimensionless_unscaled)
+            result.input_units_equivalencies = (BaseBandpass.
+                                                input_units_equivalencies)
+            return result
+
+        else:
+            print("optional dependency 'tynt' is not installed. " +
+                  "Will return 'None'")
```

You should not use un-supressable prints for something like this - in general `print` is not a reliable tool for use in a library.  Moreover, you *definitely* do not want to  return None where the user expected to get something real.  This is exactly when you should raise an exception.  So I'd say do something like:
```suggestion
            raise ImportError("optional dependency 'tynt' is not installed. To install it, do `pip install git+https://github.com/bmorris3/tynt.git@master`" )
```

Also, FYI, the `+` is unnecessary here as you originally wrote it- Python automatically concatenates multiline strings in this context.